### PR TITLE
Stop reporting admin and developer group counts

### PR DESCRIPTION
**Pull request by labkey-adam**
#### Rationale
Site Administrators and Developers groups are not so special anymore. Also, collect site role user memberships in a `HashSetValuedHashMap` to ensure unique user counts.

### CODE REVIEW

File: api/src/org/labkey/api/security/SecurityManager.java [hunk 1]
```diff
@@ -192,36 +194,28 @@ public class SecurityManager
...
-
-            Map<String, Integer> groupCounts = new HashMap<>();
-            groupCounts.put("SiteAdmin", getAllGroupMembers(getGroup(Group.groupAdministrators), MemberType.ACTIVE_USERS).size());
-            groupCounts.put("Developer", getAllGroupMembers(getGroup(Group.groupDevelopers), MemberType.ACTIVE_USERS).size());
-            result.put("SiteGroupUserCounts", groupCounts);
+            Map<String, Integer> roleCounts = mvm.keySet().stream()
+                .collect(Collectors.toMap(Role::getName, key -> mvm.get(key).size()));
 
-            return result;
+            return Map.of("SiteRoleUserCounts2", roleCounts);
```
Comment (labkey-jeckels) [inline api/src/org/labkey/api/security/SecurityManager.java]: Is this rename intentional for the final commit?

**Review by labkey-jeckels — APPROVED** [review]
Fond farewell, beautiful metrics.

File: api/src/org/labkey/api/security/SecurityManager.java (hunk 1 above)
Comment (labkey-adam) [reply api/src/org/labkey/api/security/SecurityManager.java]: Nope
